doc/faq.rst

```diff
@@ -0,0 +1,81 @@
+.. _faq:
+
+FAQ
+###
+
+Here are some frequently asked questions about the ACRN project.
+
+.. contents::
+   :local:
+
+------
+
+What hardware does ACRN support?
+********************************
+
+ACRN runs on Intel Apollo Lake and Kabby Lake boards, as documented in
+our :ref:`hardware` documentation.
+
+Clear Linux* fails to boot on my NUC
+************************************
+
+If you're following the :ref:`getting_started` documentation and the NUC
+fails to boot, here are some options to try:
+
+* Upgrade your platform to the latest BIOS
+* Verify Secure Boot is disabled in the BIOS settings:
+
+  - Press :kbd:`F2` while booting the NUC, and verify "Secure Boot" is
+    not checked in the "Boot Options"
+* Make sure you are using EFI (and not legacy BIOS)
+
+How do I configure ACRN's memory use?
+*************************************
+
+It's important that the ACRN Kconfig settings agree with the physical memory
+on your platform. Check the documentation for these option settings for
+details:
+
+* :option:`CONFIG_PLATFORM_RAM_SIZE`
+* :option:`CONFIG_SOS_RAM_SIZE`
+* :option:`CONFIG_UOS_RAM_SIZE`
+* :option:`CONFIG_HV_RAM_SIZE`
+
+For example, if memory is 32G, setup ``PLATFORM_RAM_SIZE`` = 32G
+
+::
+
+  config PLATFORM_RAM_SIZE
```

Comment:
This seems to suggest folks should actually edit the Kconfig file vs. using "make menuconfig" for example.  What is the recommend way to set one of these Kconfig options?

Reply:
@hwang37 OK, can I get a recommendation of how to update this section of the FAQ?  We don't talk much about using ``make menuconfig`` in our documentation.